cogs/minigame/minigame.py

```diff
@@ -56,15 +58,23 @@ def check_participation(m: discord.Message) -> bool:
             return m.author in self.players
 
         if self.state == GameState.STARTING:
-            embed = discord.Embed(title='Minigame Starting!', description='React below to join the game! \nThis game may contain spoilers or NSFW themes.\nPlease run `skip` in order to skip a question.', color=color)
+            embed = discord.Embed(title='Minigame Starting!', 
+                                  description='React below to join the game! \n'\
+                                              'This game may contain spoilers or NSFW themes.\n'\
+                                              'Please run `skip` in order to skip a question.', 
+                                  color=color)
             message = await ctx.send(embed=embed)
-            await message.add_reaction('🇴')
+            await message.add_reaction('🎲')
             Minigame.standby_messages.append(message.id)
             self.message_id = message.id
             for i in range(10, -1, -2):
                 players = list(map(lambda x: x.mention, self.players))
                 player_mentions = ', '.join(players)
-                embed.description = 'React below to join the game! \nThis game may contain spoilers or NSFW themes.\nPlease run `skip` in order to skip a question.\nCurrent players: {}\n{} seconds left!'.format(player_mentions, i)
+                embed.description = 'React below to join the game! \n'\
+                                    'This game may contain spoilers or NSFW themes.\n'\
+                                    'Please run `skip` in order to skip a question.\n'\
```

Comment:
what is `skip` ? is that a command `y!skip` or a subcommand `y!minigame skip`?

Reply:
To skip a question, a participating player must send `skip` as a message. There is no prefix to it, and it has been like this since the first release

Reply:
Then I would change this to 
```suggestion
                                    'Please type `skip` in order to skip a question.\n'\
```